Update Release timeline page by explaining milestones and phases (#249)
catenax-eV/catenax-ev.github.io: src/pages/timelines.md

---

### Catena-X Neptune, incl. Eclipse Tractus-X Release 26.06 & 26.09

Catena-X Neptune marks a major release introducing substantial new functionalities and standard updates across the Catena-X ecosystem.
It focuses on expanding interoperability, implementing new reference architectures, and advancing standardization efforts.
This release sets the stage for wider adoption and deeper technical alignment within both Catena-X and Tractus-X.

[![Timeline for Release 26.06](@site/static/img/timelines/cx-timeline-26.06.png)](@site/static/img//timelines/cx-timeline-26.06.png)
*Timeline for Release 26.06*

[![Timeline for Release 26.09](@site/static/img/timelines/cx-timeline-26.09.png)](@site/static/img/timelines/cx-timeline-26.09.png)
*Timeline for Release 26.09*

---

### Catena-X Proteus, incl. Eclipse Tractus-X Release 26.12 & 27.03

Proteus is another minor release and serves as the pendant  to the major release CX-Neptune.
It focuses on incremental improvements and the stabilization of the existing ecosystem.
Key objectives include refining previously introduced components, enhancing interoperability, and addressing technical feedback from earlier releases.

> This cycle ensures continuity, strengthens system quality, and lays the foundation for upcoming major milestones

Reviewer: ```suggestion
This cycle ensures continuity, strengthens system quality, and lays the foundation for upcoming major milestones.
```

*Timeline for Release 26.12*

---

### Release Management – Phases

The following section explains all phases and milestones of the Release Management Process, from initial planning to final publication.
Each phase ensures structured communication, consistent quality, and alignment between all participatings.

### Planning Phase - Milestones and Phases

The Planning Phase marks the official start of a release cycle. During this phase, the vision, objectives, and strategic direction of the upcoming release are presented and communicated to all stakeholders.
Based on this guidance, committees and expert groups begin detailed planning for the release.
<br /> This includes:

- Defining and prioritizing roadmap items,
- Breaking down roadmap items into sub-issues, and
- Identifying dependencies between items and across teams.

The Planning Phase ensures that all participating teams share a common understanding of the goals, are aligned on priorities, and have a structured plan to guide development, integration, and standardization throughout the release cycle.

| **Name** | **Category** | **Description** |
|----------|--------------|-----------------|

> | **Kick-off Release** | Milestone/Meeting | Internal communication **meeting** of the **vision** for the upcoming release. The aim is to create a **common understanding**, generate enthusiasm, and engage everyone at an early stage. |

Reviewer 2: Es ist sicher richtig, allerdings fehlt mir für jemanden neuen der Punkt "common understanding" von was?
```suggestion
| **Kick-off Release** | Milestone/Meeting | Internal communication **meeting** of the **vision** for the upcoming release. The aim is to create a **common understanding**, generate enthusiasm, and engage everyone at an early stage. |
```

"Internal communication meeting to present and align on the vision for the upcoming release. The goal is to establish a common understanding of the release scope, objectives, and priorities, and to encourage early engagement with other Expert Groups on intersecting topics and relevant dependencies.".

> | **Refinement Phase** | Phase | During this phase, the necessary people and groups collaborate to **refine the future roadmap items and features**. Roadmap item/feature authors organize appointments and communicate with the relevant contacts directly. |

Reviewer 2: Ich halte es für sinnvoll, hier eine Referenz (Hyperlink) zur Seite zu machen, wo  @stephanbcbauer die Begriffe noch mal inhaltlich erklärt hat.  Was meint ihr?

> | **Internal Roadmap Review** | Milestone/internal Meeting | The Catena-X association **review the existing roadmap items** and provide feedback on whether the items are properly defined, whether they truly qualify as roadmap items or are rather individual tasks and leave comments to ensure further refinement where necessary. |

Reviewer 3: ```suggestion
| **Internal Roadmap Review** | Milestone/internal Meeting | The Catena-X association **reviews the existing roadmap items** and provides feedback on whether the items are properly defined, whether they truly qualify as roadmap items or are rather individual tasks and leave comments to ensure further refinement where necessary. |
```

| **CX-Roadmap Item Proposal** | Milestone/Meeting | In this **meeting committee's** and **expert groups share their ideas and topics** with others. Get an **overview of topics**. **Identify and discuss dependencies** between topics. Gain insights into the overall roadmap for the whole year. |

> | **Draft Feature Freeze** | Milestone | New topics/Roadmap Items can’t be added |

Reviewer: ```suggestion
| **Draft Feature Freeze** | Milestone | New topics/Roadmap Items can’t be added. |
```

| **Alignment Day** | Milestone/Meeting | A dedicated **Tractus-X meeting** focused on **cross-team alignment and clarification of open questions** regarding specific features and their dependencies. The purpose of this meeting is solely to ensure alignment across teams, without delving into refinement or planning activities. <br /> The Alignment Day is an open Tractus-X meeting. <br /> Further information about this milestone can be found in the [official documentation](https://eclipse-tractusx.github.io/docs/oss/release-process#alignment-day). <br /> Details on how to join are available here: [Alignment Day](https://eclipse-tractusx.github.io/community/open-meetings/#one-time-meetings). |

> | **Briefing Release Planning** | Milestone/Meeting | This **meeting** is particularly valuable for those intending to contribute to **Tractus-X** or work on **Catena-X standards or normative documents** during the upcoming release. <br />The purpose of the session is to ensure participants are **well-prepared for Release Planning** by: <br /> - Providing a comprehensive overview of the overall roadmap.  <br /> - Sharing relevant organizational updates and context. <br /> This enables informed participation and alignment in the upcoming planning process. |

Reviewer 2: @ther3sa Ich halte es für nützlich, zu beschreiben, welche Zielgruppe jeder Termin hat. Wer sind Participants? 

Du hast es ja teilweise schon genannt, aber vielleicht wäre die folgende Struktur hilfreich: 

Target Group: XXX 
Description: Dein Text 
Vorbereitung: XXX

The Development Phase represents the development period of the release cycle. During this phase, teams focus on the implementation of roadmap items which can include features and standards.
<br />Key activities include:

- Designing, implementing, and testing features and components,
- Developing and refining standards and normative documents,
- Conducting integration and interoperability testing to ensure components work seamlessly within the ecosystem, and
- Preparing release candidates for subsequent testing and validation phases.

This phase is critical for ensuring that the release delivers high-quality, functional, and interoperable outputs, while maintaining alignment with strategic objectives and cross-team dependencies.

| **Name** | **Category** | **Description** |
|----------|--------------|-----------------|
| **Standard Phase** | Phase | The standard phase comprises the **development** of the **standards**.|
| **Deadline standard request**| Milestone | **Deadline** for the **submission** of standardization or **change requests** through GitHub. <br /> [Link to Change Requests](https://github.com/catenax-eV/product-standardization-prod/issues) |

> | **Quality Gate 1** | Phase| A first quality gate (“quality gate I”) **ensures** that standard candidates moving forward are **limited to those that adhere to the values and goals of the Catena-X data ecosystem**. Each standard candidate is checked for relevance, redundancy, and conformance. Provided a green light has been given, the standard candidate is officially transferred into the release schedule. The quality gate criteria are depicted in the annex |

Reviewer: ```suggestion
| **Quality Gate 1** | Phase| A first quality gate (“quality gate I”) **ensures** that standard candidates moving forward are **limited to those that adhere to the values and goals of the Catena-X data ecosystem**. Each standard candidate is checked for relevance, redundancy, and conformance. Provided a green light has been given, the standard candidate is officially transferred into the release schedule. The quality gate criteria are depicted in the annex. |
```

> | **Main Creation Phase** | Phase | In this stage, the actual **standard** is **being created in its draft form**. The moment, a standardization request has been approved, a corresponding standard template is being created and provided to the requesters. They are provided a time window in which the template must be filled out before submitting the draft standard candidate into it’s the verification phase during the current release cycle. |

Reviewer 3: ```suggestion
| **Main Creation Phase** | Phase | In this stage, the actual **standard** is **being created in its draft form**. The moment, a standardization request has been approved, a corresponding standard template is being created and provided to the requesters. They are provided a time window in which the template must be filled out before submitting the draft standard candidate into the verification phase during the current release cycle. |
```

| **Deadline for delivering standard candidate**| Milestone | **Strict deadline** for **delivering standard** documents |
| **Standards - Deadline for submitting data models** | Milestone | The deadline describes when the data model must be ready for publication in the standard; a complete pr must exist by that date.|
| **Quality Gate 2** | Phase | A second quality gate (“quality gate II”) thus **ensures** that only standard candidates that correspond to the quality expectations of Catena-X are being released. During quality gate II, the **maturity of the standard is being verified** and, provided a green light is given, a technical recommendation for its release is made to the Executive Board. The quality gate criteria are depicted in the annex. |
| **Start Opt-Out for standard candidates** | Milestone | Beginning of the  Opt-Out phase. |

> | **Opt-Out** | Phase | After a formal approval, an opt-out period of **45 days** is required, during which all association members have to be presented with the standard candidates ahead of their official release to the public. After the 45-day opt-out period has passed, the Executive Board is presented the standard candidates again for a final approval for publication, marking their official transition from being a standard candidate into an official Catena-X standard |

Reviewer: ```suggestion
| **Opt-Out** | Phase | After a formal approval, an opt-out period of **45 days** is required, during which all association members have to be presented with the standard candidates ahead of their official release to the public. After the 45-day opt-out period has passed, the Executive Board is presented the standard candidates again for a final approval for publication, marking their official transition from being a standard candidate into an official Catena-X standard. |
```

| **End Opt-Out** | Milestone |End of the Opt-Out Phase |
| **FOSS Phase** | Phase | The FOSS phase comprises the **development of the software** in the context of Tractus-X. These components are implemented by dedicated product teams, each responsible for specific functional units. Once development is complete, the phase concludes with a final integration and a comprehensive end-to-end testing process to ensure interoperability and system readiness.|

> | **Development Phase**| Phase | During this phase, teams are allocated dedicated time to **develop and contribute to FOSS components** that are part of the overall ecosystem  |

Reviewer: ```suggestion
| **Development Phase**| Phase | During this phase, teams are allocated dedicated time to **develop and contribute to FOSS components** that are part of the overall ecosystem. |
```

> | **FOSS Phase** | Phase | The FOSS phase comprises the **development of the software** in the context of Tractus-X. These components are implemented by dedicated product teams, each responsible for specific functional units. Once development is complete, the phase concludes with a final integration and a comprehensive end-to-end testing process to ensure interoperability and system readiness.|

Reviewer 2: @ther3sa, Es macht aus meiner Sicht Sinn, eine neue Tabelle anzufangen. Dann trennen wir visuell den Standard _Review_ Prozess von den folgenden Schritten.

| **Opt-In/Opt-Out of the test phase** | Milestone | This milestone marks the point at which all teams intending to participate in the upcoming release must **create** their **Release Check Issue**. By doing so, they formally indicate their **commitment** to **contribute to the release** and **provide initial information** on their **planned deliverables**. |
| **Briefing Testing** | Milestone/Meeting | The **briefing session** provides a **detailed guidance** on roles, responsibilities, and deadlines for the testing phase, ensuring that everyone involved understands their tasks and deliverables. <br /> The Briefing Testing is an open Tractus-X meeting. <br /> Details and access can be found here: [Briefing Testing](https://eclipse-tractusx.github.io/community/open-meetings/#one-time-meetings).  |

> | **Feature Freeze** | Milestone | The Feature Freeze marks a milestone in the release process, at which point **no further development** of new features or functionality is permitted. This ensures system stability and allows teams to focus exclusively on integration, testing, and defect resolution in preparation for the release. |

Reviewer 3: ```suggestion
| **Feature Freeze** | Milestone | The Feature Freeze marks a milestone in the release process, at which point **no further development** of new features or functionality is permitted for this release. This ensures system stability and allows teams to focus exclusively on integration, testing, and defect resolution in preparation for the release. |
```

> | **Int Deployment** |Phase | Release candidate versions deployed on INT |

Reviewer: ```suggestion
| **Int Deployment** |Phase | Release candidate versions deployed on INT. |
```

> | **Int Deployment** |Phase | Release candidate versions deployed on INT |

Reviewer 3: ```suggestion
| **INT Deployment** |Phase | In this phase, the release candidates have to be deployed on INT. |
```

> | **Kick-Off Testing** | Milestone/Meeting | Meeting on start of E2E test phase in regular daily slot at 8:30 – 9:00 a.m. <br /> The Kick-Off Testing is an open Tractus-X meeting. <br /> Details and access can be found here: [Kick-Off Testing](https://eclipse-tractusx.github.io/community/open-meetings/#one-time-meetings). |

Reviewer 3: ```suggestion
| **Kick-Off Testing** | Milestone/Meeting | Meeting at the start of E2E test phase in the regular daily slot at 8:30 – 9:00 a.m. <br /> The Kick-Off Testing is an open Tractus-X meeting. <br /> Details and access can be found here: [Kick-Off Testing](https://eclipse-tractusx.github.io/community/open-meetings/#one-time-meetings). |
```

> | **Testing Phase** | Phase | - Tests executed and status reported consistently. <br /> - Afterwards Retrospective meeting for test phase|

Reviewer 3: ```suggestion
| **Testing Phase** | Phase | In this phase, tests are executed and status is reported consistently by the product teams in daily meetings from 8:30 - 9:00 a.m. CET <br /> |
```

<br /> Key activities include:

- Final review of all roadmap items, sub-issues, and standardization outputs,
- Completion of end-to-end testing and resolution of any remaining defects,
- Preparation of release notes, changelogs, and documentation, and
- Conducting a release recap or retrospective, summarizing achievements, lessons learned, and insights for future cycles.

This phase ensures transparency, traceability, and quality assurance, providing confidence to all participating teams and external partners that the release meets the agreed standards and objectives.

| **Name** | **Category** | **Description** |
|----------|--------------|-----------------|
| **Release Finalization** | Phase | During this phase, all necessary activities to ensure a smooth and transparent release are carried out. For Tractus-X, this includes the preparation of the **changelog** and a **review** of the **Release Check Issues**. Any outstanding questions are clarified directly with the respective product teams. In the Catena-X context, detailed Release Notes are created, and the Deployment Gate is executed to formally authorize the release for deployment. |

> | **Publish Release** | Milestone | The release will be published |

Reviewer: ```suggestion
| **Publish Release** | Milestone | The release will be published. |
```
